Tests/BootCamp.Chapter.Tests/TestsHand.cs

```diff
@@ -0,0 +1,34 @@
+﻿using BootCamp.Chapter.Gambling;
+using System.Collections.Generic;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class TestsHand : Gambling.Hand
+    {
+        //PS cant remove cards from hand? weird....
```

Comment:
The intention was to have a Poker hand. Poker hand does not need removing cards. You just add cards one by one, at the end of the game reveal them, and then start over with new cards.